docs/src/introduction/sway_quickstart.md

```diff
@@ -196,46 +193,11 @@ Logs:
 TransactionId(HexFormatted(7cef24ea33513733ab78c5daa5328d622d4b38187d0f0d1857b272090d99f96a))
 ```
 
-Note the contract ID—you will need it if you want to build out a frontend to interact with this contract.
+Note the contract ID — you will need it if you want to build out a frontend to interact with this contract.
 
-## Testing your Contract
+## Testing Your Contract
 
-In the directory `tests`, navigate to `harness.rs.` Here you'll see there is some boilerplate code to help you start interacting with and testing your contract.
-
-At the bottom of the file, define the body of `can_get_contract_instance`. Here is what your code should look like to verify that the value of the counter did get incremented:
-
-```sway
-#[tokio::test]
-async fn can_get_contract_instance() {
-    // Increment the counter
-    let _result = instance.increment().call().await.unwrap();
-
-    // Get the current value of the counter
-    let result = instance.counter().call().await.unwrap();
-    assert!(result.value > 0);
-}
-```
-
-Run the following command in the terminal: `forc test`.
-
-You'll see something like this as your output:
-
-```console
-  Compiled library "core".
-  Compiled library "std".
-  Compiled contract "counter_contract".
-  Bytecode size is 224 bytes.
-   Compiling counter_contract v0.1.0 (<path/to/counter_contract>)
-    Finished test [unoptimized + debuginfo] target(s) in 4.55s
-     Running tests/harness.rs (target/debug/deps/integration_tests-7a2922c770587b45)
-
-running 1 test
-test can_get_contract_id ... ok
-
-test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.09s
-```
-
-Congratulations, you've just created and tested your first Sway smart contract 🎉.
+We will cover how to test your contract later but, if you are eager to take a look, see [Testing with Rust](../testing/testing-with-rust.md).
```

Comment:
Maybe it would make sense to link both testing with rust and unit testing?

Reply:
Hey @kayagokalp! Unit testing shows a 404 now.

<img width="674" alt="Screenshot 2022-11-21 at 10 06 40 PM" src="https://user-images.githubusercontent.com/8782666/203148605-c54cd453-a168-49ab-a921-7438db733908.png">

Reply:
Yes this was fixed on the `master` book so it should be okay now.
https://fuellabs.github.io/sway/master/testing/unit-testing.html

Reply:
Whoops. Added now, see the commit below.